lib/content-services/src/lib/search/components/search-filter-tabbed/search-filter-tabbed.component.ts

```diff
@@ -29,12 +29,39 @@ import { TranslateModule } from '@ngx-translate/core';
     styleUrls: ['./search-filter-tabbed.component.scss'],
     encapsulation: ViewEncapsulation.None
 })
-export class SearchFilterTabbedComponent {
+export class SearchFilterTabbedComponent implements OnInit, OnDestroy {
     @ContentChildren(SearchFilterTabDirective)
     tabsContents: QueryList<SearchFilterTabDirective>;
 
     selectedIndex: number = 0;
 
+    @ViewChild(MatTabGroup)
+    private readonly tabGroup: MatTabGroup;
+
+    private readonly intersectionObserver = new IntersectionObserver(
+        (entries) => {
+            if (!entries[0].isIntersecting) {
+                this.tabGroup.selectedIndex = (this.selectedIndex + 1) % this.tabsContents.length;
```

Comment:
Why do we need to set `this.tabGroup.selectedIndex` twice?

Reply:
This is some angular material problem when we use tabs together with some kind of overlaps like on below screenshot. Basically the problem is following: 
When we close this popup then seems like Tabs component becomes broken because after reopening popup Tabs component doesn't know on which tab it was before closing so when we try to switch tab we can see for a moment strange text sliding effect. 
To resolve that problem I wrote code which observes for hiding/showing component and I do single switching of tabs so Tabs component can restore it's state. It seems like only solution for that problem, because we can't just "destroy" tabs component and recreate it because we need to keep previous state. 

<img width="734" alt="image" src="https://github.com/user-attachments/assets/92b1426e-7631-4f21-a55d-184da96e4470">